tests/get_binary_test.ts

```diff
@@ -9,7 +9,15 @@ import { assertStringIncludes } from "https://deno.land/std@0.201.0/assert/asser
 Deno.env.set("ASTRAL_QUIET_INSTALL", "true");
 const cache = await Deno.makeTempDir({ prefix: "astral_test_get_binary" });
 const permissions = {
-  write: [cache],
+  write: [
+    cache,
+    // Chromium lock on Linux
+    `${Deno.env.get("HOME")}/.config/chromium/SingletonLock`,
+    // Chromium lock on MacOS
+    `${
+      Deno.env.get("HOME")
+    }/Library/Application Support/Chromium/SingletonLock`,
```

Comment:
I'm not seeing one for windows, does this issue not happen on windows?

Reply:
I don't think so, it doesn't die on github runner, and I never encoutered this issue on my Windows too

Reply:
Interesting, good to note.

Reply:
Reading more about this, it seems to be due the way [Windows locks files](https://en.wikipedia.org/wiki/File_locking#In_Microsoft_Windows). Apparently just opening files on Windows locks the file, unless the apps flags them explicitely as shareable, so my guess it's why there's no SingletonLock file: it's already implicit in Windows